docs/sources/contributing/devenvironment.md

```diff
@@ -126,11 +126,13 @@ something like this
     PASS
     ok      github.com/docker/docker/utils        0.017s
 
-If $TESTFLAGS is set in the environment, it is passed as extra arguments
+If `$TESTFLAGS` is set in the environment, it is passed as extra arguments
```

Comment:
Agreement error. Should probably be "If `$TESTFLAGS` is set in the environment, it will pass extra arguments..."

(I know the error isn't yours, but while we're in here...)